scripts/install

```diff
@@ -259,10 +259,8 @@ ynh_multimedia_addaccess $app
 # Fix app ownerships & permissions
 chown -R $app:www-data "$install_dir"
 chown -R $app: "$data_dir"
```

Comment:
On my server, the main issue is this line. Idk how to speed up this chown though

Reply:
Quick look on the internet : https://unix.stackexchange.com/questions/245274/chown-only-where-needed-speedup-chown

Apparently using `find` to only apply `chown` for the relevant file is way faster, could be nice to give it a try

Reply:
If you have a lot of files, the find method could trigger a message saying there is too many destination listed (if i remeber an old bug)

Reply:
It seems `chown -R $app: $data_dir` is performed 3 times during upgrade, which shouldn’t be necessary because `$data_dir` is not touched during the upgrade as it is in another place than `$install_dir` ? (cf. https://docs.nextcloud.com/server/latest/admin_manual/maintenance/manual_upgrade.html)